.pre-commit-hooks.yaml

```diff
@@ -1,16 +1,49 @@
--   id: apigentools-validate
-    name: Validate OpenAPI specification files
-    language: python
-    entry: apigentools
-    args:
-      - validate
-    description: Validate OpenAPI specification files generated by apigentools using configuration inside this repository.
-    pass_filenames: false
--   id: container-apigentools-validate
-    name: Validate OpenAPI specification files using container wrapper
-    language: python
-    entry: container-apigentools
-    args:
-      - validate
-    description: Validate OpenAPI specification files generated by apigentools using configuration inside this repository using container wrapper.
-    pass_filenames: false
+- id: validate
+  name: Validate OpenAPI specification files
+  language: python
+  entry: apigentools validate
+  description: Validate OpenAPI specification files generated by apigentools using configuration inside this repository.
+  pass_filenames: false
+- id: templates
+  name: Run apigentools templates
+  language: python
+  entry: apigentools templates
+  description: Run apigentools templates inside this repository.
+  pass_filenames: false
+  always_run: true
+  stages: [manual]
+- id: generate
+  name: Run apigentools generate
+  language: python
+  entry: apigentools generate
+  description: Run apigentools generate inside this repository.
+  pass_filenames: false
+  always_run: true
+  stages: [manual]
+- id: test
+  name: Run apigentools test
+  language: python
+  entry: apigentools test
+  description: Run apigentools test inside this repository.
+  pass_filenames: false
+  always_run: true
+  stages: [manual]
+- id: push
+  name: Run apigentools push
+  language: python
+  entry: apigentools push
+  description: Run apigentools push inside this repository.
+  pass_filenames: false
+  always_run: true
+  stages: [manual]
+- id: apigentools
```

Comment:
whats this one for? Just debugging?

Reply:
yes. and also one can define custom command easily using `args`